src/azure-cli/setup.py

```diff
@@ -98,7 +98,7 @@
     'azure-mgmt-managementgroups~=0.1',
     'azure-mgmt-maps~=0.1.0',
     'azure-mgmt-marketplaceordering~=0.1',
-    'azure-mgmt-media~=2.1,>=2.1.0',
+    'azure-mgmt-media~=3.0,>=3.0.0',
```

Comment:
```suggestion
    'azure-mgmt-media~=3.0',
```
I think `~=3.0` already includes `>=3.0.0`.